recipes/dataframe/all/conanfile.py

```diff
@@ -65,7 +70,7 @@ def configure(self):
             "Visual Studio": "15",
             "gcc": "7",
             "clang": "6",
-            "apple-clang": "9.0",
+            "apple-clang": "10.0" if tools.Version(self.version) >= "1.12.0" else "9.0",
```

Comment:
Why does apple-clang changes? C++ feature support or standard support?

Reply:
https://github.com/conan-io/conan-center-index/pull/3195#issuecomment-713535531 upstream requires higher STL support

Reply:
As per @prince-chrismc , in version "1.12" I started using `std::any` and for some reason, apple-clang "9.0" could not find header file `any`